sparse/numba_backend/_coo/common.py

```diff
@@ -1456,7 +1456,7 @@ def _compute_minmax_args(
             if not found:
                 result_data.append(current_coord + 1)
 
-    return (result_indices, result_data)
+    return (result_indices, np.array(result_data, dtype=np.intp))
```

Comment:
Let's avoid a copy if we can?

```suggestion
    return (result_indices, np.asarray(result_data, dtype=np.intp))
```

Reply:
With the current implementation there's no way to avoid a copy.
The `result_data` that we pass to `np.array` is defined as:
```python
result_data = []
```
and then individual coords are added with `result_data.append(...)` in a loop.
So `result_data` can't be a view of another array. It's a Python list that is passed to NumPy (always a copy is made for Python lists), therefore `np.array` is fine here.